jobs/jobs/utils.py

```diff
@@ -788,6 +786,39 @@ async def get_annotation_revisions(
     return response
 
 
+async def get_annotations_by_revisions(
+    current_tenant: Optional[str], jw_token: str, revisions: List[str]
+) -> Optional[Dict[str, Any]]:
+    """Get annotations by filtering"""
+
+    headers = {
+        "X-Current-Tenant": current_tenant,
+        "Authorization": f"Bearer: {jw_token}",
+    }
+
+    post_data = {
+        "filters": [
+            {"field": "revision", "operator": "in", "value": revisions}
+        ]
+    }
+
+    try:
+        _, response = await fetch(
+            method="POST",
+            url=f"{ANNOTATION_SERVICE_HOST}/annotation",
+            headers=headers,
+            body=post_data,
+        )
+    except aiohttp.client_exceptions.ClientError as err:
+        logger.exception("Failed request to get annotations by revisions")
+        raise fastapi.HTTPException(
+            status_code=fastapi.status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=f"Failed request to the Annotation Manager: {err}",
```

Comment:
I think 422 is not a correct error code for that. I'd set it to 400. And also do not expose internal exception to outside. Consider http exception as an end user message. User has no idea what is an annotation manager.